setup.py

```diff
@@ -154,7 +154,7 @@ def run(self):
         'numpy>=1.19.2',
         'omegaconf==2.0.6',
         'openpyxl>=3.0.7',
-        'pandas>=1.1.5',
+        'pandas==1.1.5',
```

Comment:
seems this line change is also merged in main?

Reply:
oh, if so, we don't need this pr